packages/nimble-components/src/table/tests/table-selection.spec.ts

```diff
@@ -6,12 +6,16 @@ import { type Fixture, fixture } from '../../utilities/tests/fixture';
 import {
     TableRecord,
     TableRowSelectionEventDetail,
-    TableRowSelectionMode
+    TableRowSelectionMode,
+    TableRowSelectionState
 } from '../types';
 import { TablePageObject } from './table.pageobject';
+import type { TableColumnText } from '../../table-column/text';
 
 interface SimpleTableRecord extends TableRecord {
+    id?: string;
     stringData: string;
+    stringData2: string;
 }
 
 const simpleTableData = [
```

Comment:
Feel free to ignore, but I got thrown (despite understanding that the tests would fail if not set up right) by the fact that we were setting the `idFieldName` to `stringData` in various tests, since in the diff I saw that the `groupableTableData` had an `id` field and a `stringData` field that didn't not have unique values across all records. It might be beneficial to change the name of `stringData` in `simpleTableData` to be `id`.